src/CommunityToolkit.Maui.Core/Essentials/SpeechToText/ISpeechToText.shared.cs

```diff
@@ -27,24 +27,15 @@ public interface ISpeechToText : IAsyncDisposable
 	/// </summary>
 	SpeechToTextState CurrentState { get; }
 
-	/// <summary>
-	/// Converts speech to text in real time.
-	/// </summary>
-	/// <param name="culture">Speak language</param>
-	/// <param name="recognitionResult">Intermediate recognition result.</param>
-	/// <param name="cancellationToken"><see cref="CancellationToken"/></param>
-	/// <returns>Final recognition result</returns>
-	Task<SpeechToTextResult> ListenAsync(CultureInfo culture, IProgress<string>? recognitionResult, CancellationToken cancellationToken = default);
-
 	/// <summary>
 	/// Starts the SpeechToText service
 	/// </summary>
 	/// <remarks>
 	/// Real time speech recognition results will be surfaced via <see cref="RecognitionResultUpdated"/> and <see cref="RecognitionResultCompleted"/>
 	/// </remarks>
-	/// <param name="culture">Speak language</param>
+	/// <param name="options"><see cref="SpeechToTextOptions"/></param>
 	/// <param name="cancellationToken"><see cref="CancellationToken"/></param>
```

Comment:
What will the `cancellationToken` parameter do if it is canceled? Will it stop listening and result in a non-successful recognition result? If so I think we should add this detail to the xml docs.

Reply:
No, Cancellation Token will abort StartListening operation. If listening is started you can only cancel it using StopListening()

Reply:
Thanks I have added some xml comment improvements